### DISCUSSION TITLE
Is there a disabled state for an anchor?

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#1637] [Q&A] [answered]
I'm designing an updated flow for the Authy app and we use the anchor to lead to a full page modal that explains twilios data privacy in simple context for the end user. We have a loading state for this screen - but I noticed we didn't have a disabled state for the anchor in any PASTE documentation or figma libraries. 

Do we really not have a disabled state for an anchor?

<img width="334" alt="Screen Shot 2021-07-02 at 1 13 16 PM" src="https://user-images.githubusercontent.com/78432794/124313971-48714880-db37-11eb-9d68-c7f040c1caef.png">

**Comment by nkrantz** [comment, marked as answer]
Hey, @mjanssen2021!

Updating my previous answer with a bit more context:

We don't have a disabled state for the Anchor component, but in general we recommend using Buttons to launch Modals. The distinction between Buttons and Anchors informs your customers (especially those who are using assistive technology) of what they can expect to happen when they interact with the component. Anchors should open pages, while Buttons should perform actions. 

URLs don't have a disabled state in Paste because it's not a supported interaction in general (there's no disabled state for the HTML `a` tag). There are hack-y ways to get around disabling an anchor, but it can be confusing for users. 

On the other hand, Buttons can be disabled. We have a [Link-Style Button](https://paste.twilio.design/components/button/#link-style-button) that looks like a Link, but still communicates to assistive technology users that clicking on it will not take them to a new page. This would probably be your best bet for this use case.

Thanks!
